src/utils/formatters/formatCurrencyWithSymbol.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+// How to format local currency
+// https://developers.shopware.com/blog/2017/02/06/currency-formatting-is-easy-isnt-it/
+
+import {
+  toBigNumber,
+  type ToBigNumberValue,
+} from 'utils/numbers/toBigNumber'
+
+function formatCurrencyWithSymbol(
+  amount: ToBigNumberValue,
+  currencyCode: string,
+  locale: string = 'en-US',
+): string {
+  const opts = {
+    style: 'currency',
+    currency: currencyCode,
+  }
+
+  const numberAmount: number = toBigNumber(amount).toNumber()
+
+  const numberFormat = new Intl.NumberFormat(locale, opts)
```

Comment:
so, it should be added in `browsercheck`

Reply:
I think better solution is to add safari  polyfill for it. But now this browser is not supported

Reply:
Now, I added fallbacks if amount or locale or Intl.NumberFormat are not valid